administrator/components/com_plugins/tmpl/plugin/edit.php

```diff
@@ -32,11 +32,11 @@
 <form action="<?php echo Route::_('index.php?option=com_plugins&view=plugin&layout=' . $layout . $tmpl . '&extension_id=' . (int) $this->item->extension_id); ?>" method="post" name="adminForm" id="style-form" aria-label="<?php echo Text::_('COM_PLUGINS_FORM_EDIT'); ?>" class="form-validate">
 	<div class="main-card">
 
-		<?php echo HTMLHelper::_('uitab.startTabSet', 'myTab', array('active' => 'general')); ?>
+		<?php echo HTMLHelper::_('uitab.startTabSet', 'myTab', ['active' => 'general', 'recall' => true, 'breakpoint' => 768]); ?>
 
 		<?php echo HTMLHelper::_('uitab.addTab', 'myTab', 'general', Text::_('COM_PLUGINS_PLUGIN')); ?>
 
-		<div class="row mt-2">
+		<div class="row mt-2 ps-4">
```

Comment:
```suggestion
		<div class="row mt-2">
```

![34813-plugin](https://user-images.githubusercontent.com/368084/126847435-b4eca67e-2b14-42d8-a0de-57a51256fb07.jpg)